src/guacd/systemd/guacd.service.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Guacamole Server
+Documentation=man:guacd(8)
+
+[Service]
+User=daemon
+ExecStart=@sbindir@/guacd -f
```

Comment:
Is this the generally-accepted best practice? Relying on systemd to handle the daemonization for the process, management of any PID file, etc.?

Reply:
Yes - most of what I've seen in systemd unit files is that you put the process in foreground mode and allow systemd to manage it.  I can't cite a source that says it, just my anecdotal experience.

Reply:
Googling around a bit, definitely seems the way of things for systemd: http://www.mikeperham.com/2014/09/22/dont-daemonize-your-daemons/